src/go/client.go

```diff
@@ -0,0 +1,66 @@
+package ifsc
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+)
+
+const API_BASE = "https://ifsc.razorpay.com"
+
+type IFSCResponse struct {
+	Bank     string `json:"BANK"`
+	Branch   string `json:"BRANCH"`
+	Address  string `json:"ADDRESS"`
+	Contact  string `json:"CONTACT"`
+	City     string `json:"CITY"`
+	District string `json:"DISTRICT"`
+	State    string `json:"STATE"`
+	IFSC     string `json:"IFSC"`
+	BankCode string
+}
```

Comment:
Q: What happens if the API response changes?

Reply:
if the key's change, then it would fail if we don't update it. Also, the default json tags support only one tag name. 
We can  keep IFSCResponse as an `interface{}`. But setting bank code would be an issue since if the API response changes, we would have to make change in `setBankCode()` again.
Also, if we keep it as `interface{}` , we can't access the individual fields directly. We would have to either unmarshal it into:
1. a struct again, or
2. a map[string]string 

to access the fields.